packages/client/hmi-client/src/components/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-ciemss-drilldown.vue

```diff
@@ -40,11 +40,11 @@
 											<td>
 												<tera-input-text v-model="ele.newName" auto-focus class="w-full" placeholder="Add a name" />
 											</td>
-											<td v-for="(row, indx) in ele.modelConfigurationMappings" :key="indx">
+											<td v-for="key in Object.keys(ele.modelConfigurationMappings)" :key="key">
```

Comment:
`Object.keys(...)` does not guarantee order, does it matter here or do we need things to be consistent? 

It might mean that if current current order is `[a, b, c]` and you added `d`, it might become "[a, d, b, c]" instead of "[a, b, c, d]"

Reply:
Great catch! Thank you
This may have caused user confusion
I will update this

Reply:
I am not using `modelConfigurationIds` for this which is a list of the model configs from the inputs